### PR TITLE
TE-8042 / TE-7945 Replaced Travis with GA

### PR DESCRIPTION
**Pull request by Nidhognit**
Task https://spryker.atlassian.net/browse/TE-8042

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  - APPLICATION_STORE=DE
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.4'
```
Comment (dereuromark) [inline .github/workflows/ci.yml:20]: always min/max

Comment (Nidhognit) [reply .github/workflows/ci.yml:20]: Maybe i understand it wrong, but for 7.3 we have separate "prefer-lowest" no?

Comment (dereuromark) [reply .github/workflows/ci.yml:20]: No, prefer lowest uses the same min, but additionally also min dependencies (not max dependencies).

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -17,7 +17,8 @@ jobs:
       fail-fast: false
       matrix:
         php-version: [
-            '7.4'
+            '7.4',
```
Comment (dereuromark) [inline .github/workflows/ci.yml:20]: Min/max order

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -91,13 +87,12 @@ jobs:
...
             key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-            restore-keys: |
-            ${{ runner.os }}-composer-
+            restore-keys: ${{ runner.os }}-composer-
 
       - name: Composer prefer-lowest
         run: |
-            composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable
             composer require --dev dereuromark/composer-prefer-lowest
+            composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable
```
Comment (dereuromark) [inline .github/workflows/ci.yml:95]: The order was correct before

**Review by dereuromark — APPROVED** [review]

**Pull request merged**